cypress/integration/navigation.spec.js

```diff
@@ -0,0 +1,61 @@
+context('Navigation', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
```

Comment:
A baseUrl for `cy.visit` can be set in `cypress.json`. Makes the tests more robust.

Reply:
Done, but I still need to pass the url, which confuses. If I understood the docs correctly this shouldn't be neccessary? 
cypress.json: `{"baseUrl": "http://localhost:3000/"}`
navigation.spec.js:
```
const url = Cypress.config().baseUrl
context('Navigation', () => {
  beforeEach(() => {
    cy.visit(url)
  })
```

Reply:
Whatever you pass to `cy.visit` is appended to the base url, I don't know if you can leave it empty. Or simply do `cy.visit("/")` for the root path.

Reply:
Absolutely, you are right. Works. Empty caused an error, but "/" works